src/search/searchReducer.js

```diff
@@ -112,25 +112,25 @@ function* initialSearch() {
     let state = yield select();
     try {
         if (!state.search.initialSearchDone) {
-            // For å få tak i alle tilgjengelige fasetter (yrke, område osv), så gjør vi først
-            // et søk uten noen søkekriterier. Dette vil returnere alle kjente fasettverdier på
-            // tverss av alle annonsene i backend. Vi henter også all geografi info fra server.
-            let searchRes = yield call(fetchSearch, {});
-            const locationRes = yield call(fetchLocations);
+            // Do all 3 required calls in one yield all
+            const calls = [
+                call(fetchSearch, {}),
```

Comment:
Her ville jeg lagt til kommentar:
`call(fetchSearch, {}), // Må gjøre et tomt søk for å finne alle mulige søkekriterier`